banderwagon/element.go

```diff
@@ -394,7 +385,8 @@ func (p *Element) IsOnCurve() bool {
 	return point_aff.IsOnCurve()
 }
 
-// IsIdentity returns true if p is the identity element.
+// Normalize returns normalizes a point to affine form.
+// If the point is at infinity, returns an error.
```

Comment:
To close https://github.com/crate-crypto/go-ipa/issues/65

Reply:
Should this read: "Normalize returns a point in affine form" ?

Reply:
Yep, I'll change.

Reply:
Done.

Reply:
Not a blocker; in a future Pr, we should likely stick to american english and do Normalize and remove Normalise

Reply:
I'll change it now, should be easy.